packages/swagger/src/swagger-controller.spec.ts

```diff
@@ -311,7 +311,46 @@ describe('SwaggerController', () => {
         const expected = readFileSync(join(__dirname, 'index.primary.spec.html'), 'utf8');
         strictEqual(response.body, expected);
 
+      });
+    it('should properly render the template given options are { url: "xxx" } and uiOptions are { docExpansion:"none" }', async () => {
```

Comment:
Could you make the two new tests to have the same indentation as the others? In VSCode, you can select them and press just once `tab`.